docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx

```diff
@@ -73,6 +73,12 @@ To add new Waypoints to a mission while edit mode is enabled select the
 locations where the user clicks on the map. These will appear as red 
 Waypoints with the exception of the first waypoint (green) and the last waypoint (yellow).
 
+:::note
+
+**The first Waypoint in the mission must be within 1.5 meters of the platforms current position.**

```

Comment:
platforms -> UGV's
I'm not opposed to "platform", but it seems inconsistent with the rest of the manual where we use UGV.